docs/remote.md

```diff
@@ -49,6 +49,16 @@ When loaded, a remote driver process receives an HTTP POST on the URL `/Plugin.A
 
 Other entries in the list value are allowed; `"NetworkDriver"` indicates that the plugin should be registered with LibNetwork as a driver.
 
+### Set capability
+
+After Handshake, the remote driver will receive another POST message to the URL `/NetworkDriver.GetCapability` with no payload. The driver's response should have the form:
```

Comment:
@WeiZhang555 I think @mapuri suggests you to change the endpoint name to `/NetworkDriver.GetCapabilities` (notice the plural form of GetCapability)